<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlankTool</class>
 <widget class="QMainWindow" name="BlankTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="main_widget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9,0">
     <item>
      <layout class="QHBoxLayout" name="main_title_bar"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="main_content_layout" stretch="1,1,1">
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="aab_bar_layout">
         <property name="leftMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="apk_bar_layout"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="other_bar_layout"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
